worldedit-forge/src/main/resources/mcmod.info

```diff
@@ -14,7 +14,8 @@
         "Forge@[${forgeVersion},)"
     ],
     "dependencies": [
-        "Forge@[${forgeVersion},)"
+        "Forge@[${forgeVersion},)",
+        "ForgeMultipart"
```

Comment:
FMP is not part of forge, and yes, the server will crash if you install WE without installing FMP this way,